src/less/mixins/icon/base/icon-mixins.less

```diff
@@ -454,7 +454,28 @@
 @icon-video-pause-width: 24px;
 @icon-breadcrumb-width: 8px;
 @icon-breadcrumb-height: 8px;
-
+@icon-0-star-width: 64px;
```

Comment:
I'm pretty sure these are always going to be the same width and height, right? So we can collapse these down to just two variables: `icon-star-rating-width` and `icon-star-rating-height`